desktop/src/main/java/org/destinationsol/desktop/SolDesktop.java

```diff
@@ -11,55 +26,96 @@
 import org.slf4j.LoggerFactory;
 import org.terasology.crashreporter.CrashReporter;
 
-import java.io.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
 import java.nio.charset.Charset;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.Date;
 import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * This class is the desktop entry point for the whole DestinationSol application. It handles the creation and launching
```

Comment:
You might want to clarify "desktop" to "desktop (PC) application" as opposed to a mobile application